tests/renderButtons.test.ts

```diff
@@ -5,7 +5,7 @@ import { renderButtons } from '../src/renderButtons';
 
 function expectIcon(button: HTMLDivElement, text: string) {
     let imageCount = 0;
-    for (const child of button.children) {
+    for (const child of <any>button.children) {
```

Comment:
Does the for...of work in IE11? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of is saying it's unsupported.

Reply:
This is test code that runs in node via `yarn test`. The change to include `<any>` is actually a side effect of downgrading to `es5`. So it's an indirect consequence of the IE11 fixes.